cmd/mdatagen/metric-metadata.yaml

```diff
@@ -22,6 +24,8 @@ attributes:
     name_override:
     # Required: description of the attribute.
     description:
+    # Optional: Whether the attribute is emitted by default.
+    enabled: <bool>
```

Comment:
Let's not add it to the metric attributes yet. That will be done as part of https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/16374